adding_species_manuscript.tex

```diff
@@ -209,23 +209,21 @@ \section*{Introduction}
 For instance, simulations provide training data
 for inference methods based on machine learning \citep{Schrider2018} and
 Approximate Bayesian Computation \citep{Csillery2010}. They can also serve as
-baselines for further analyses: for example, simulations incorporating
```

Comment:
I would keep "for example," because there are potentially other ways they could be used as "baselines."